chain-impl-mockchain/src/block/builder.rs

```diff
@@ -66,24 +66,24 @@ impl BlockBuilder {
 
     /// set a transaction in the block to build
     ///
-    /// Equivalent to call `block_builder.message(Message::Transaction(transaction))`
+    /// Equivalent to call `block_builder.message(Fragment::Transaction(transaction))`
     pub fn transaction(
         &mut self,
         signed_transaction: AuthenticatedTransaction<Address, NoExtra>,
     ) -> &mut Self {
-        self.message(Message::Transaction(signed_transaction))
+        self.message(Fragment::Transaction(signed_transaction))
     }
 
     /// add a message in the block to build
-    pub fn message(&mut self, message: Message) -> &mut Self {
+    pub fn message(&mut self, message: Fragment) -> &mut Self {
```

Comment:
Should this method and `messages` below be also renamed?

Reply:
yes, but I don't want to do too many changes at once